scripts/setup.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export PATH="${PATH}:$(yarn global bin)"
```

Comment:
Lets make sure we add a comment to be consistent with what we've done in other repos. We should be indicating why this line is required.

```bash
# Add yarn to the $PATH so cli commands do not fail
export PATH="${PATH}:$(yarn global bin)"
```